cli/internal/cmd/configfetchmeasurements.go

```diff
@@ -79,6 +88,46 @@ func configFetchMeasurements(cmd *cobra.Command, fileHandler file.Handler, clien
 	return nil
 }
 
+func verifyWithRekor(cmd *cobra.Command, hash string) error {
+	r, err := sigstore.NewRekor()
+	if err != nil {
+		cmd.PrintErrln("Unable to construct Rekor client.")
+		return err
+	}
+
+	uuids, err := r.SearchByHash(cmd.Context(), hash)
+	if err != nil {
+		cmd.PrintErrln("Unable to find hash in Rekor.")
+		return err
```

Comment:
```suggestion
		return fmt.Errorf("searching Rekor for hash: %w", err)
```
Is there a reason to go against our current pattern. IMO the information still work inside the error. Do we want to be more explicit for security errors?